src/presentational/DialogScreen-Download.jsx

```diff
@@ -5,6 +5,7 @@ import {saveAs} from 'browser-filesaver';
 export default class DialogScreen_DownloadCSV extends DialogScreen {
   constructor(props) {
     super(props);
```

Comment:
`super()` must be used before the `this` keyword can be used, so I think it should stay in.